### DISCUSSION TITLE
Stage M2 + Thèse - Neural Methods for Plasma

### DISCUSSION THREAD

**Discussion by jaghili** [master-csmi/csmi#26] [Stages et Emplois]
Offre de stage à l'IRMA, encadrée par E. Franck (INRIA), Stanislas Pamela (UKAEA, Culham), et moi-même. 

Le sujet porte sur les méthodes Neural Galerkin pour des applications en physique des plasmas : https://recrutement.inria.fr/public/classic/fr/offres/2023-07017

Poursuite en thèse possible (et souhaitée !) et déplacements à Culham (Royaume Uni) éventuels pendant la thèse.

**Comment by Kaoutar2Aqil** [comment]
Au cours de mes études à la Faculté de Dhar El Mehraz a Fès, j'ai développé un fort intérêt pour le domaine de l'apprentissage automatique et l'innovation technologique. Mon expérience en tant que membre actif d'un club universitaire m'a permis de participer à plusieurs compétitions, renforçant ainsi mon esprit d'équipe et mes compétences en résolution de problèmes. De plus, j'ai eu l'opportunité de recevoir des lettres de recommandation de mes professeurs, attestant de mon engagement et de mes aptitudes académiques. Ces expériences m'ont préparé à relever les défis de votre programme et à y contribuer de manière significative.
